### ISSUE TITLE
[Feature request] Add "Do not show again" to the "Update Available" dialogue window

### ISSUE THREAD

**Issue by davidhedlund** [RPCS3/rpcs3#14742]
### Quick summary

A checkbox ("Do not show again"), in the "Update Available" dialogue window would be useful if you encounter regression bugs, and need to use an old version. If you are asked to press a yes button every time you open the emulator, there's a risk that you accidentally will update the system.

### Details

"Check for updates on startup" exist, but it's moderately difficult to find it (I didn't find it when I opened this issue for example):

![image](https://github.com/RPCS3/rpcs3/assets/6416073/5cc3bc06-41dd-44e8-a768-36fcc4b98c4b)

Proposed (purple area = image manipulated) "Do not show again"[1] check box:

![image](https://github.com/RPCS3/rpcs3/assets/6416073/ea5a8acb-d23a-49eb-a129-0a60224cbcb7)

# References
1: "Do not show again" option exists in the "Welcome to RPCS3" dialogue window:

![image](https://github.com/RPCS3/rpcs3/assets/6416073/cf1caaa9-0b38-4ee9-9ff3-fe4b1c138a24)

**Comment by FlexBy420** [comment]
You can already disable that in GUI settings

**Comment by davidhedlund** [comment]
> You can already disable that in GUI settings

Where specifically? I have look around for at least 10 minutes, but I didn't find anything.

**Comment by davidhedlund** [comment]
@elad335 It would still be useful to add "Do not show again" directly to the "Update Available" dialogue window.

**Comment by davidhedlund** [comment]
Thank you very much @elad335

**Comment by davidhedlund** [comment]
> You can already disable that in GUI settings

@elad335 Can you please help me find it?

**Comment by davidhedlund** [comment]
> You can already disable that in GUI settings

Thank you. I updated the top post.

**Comment by elad335** [comment]
This probably requires an additional functionality such as an "update" button in the main toolbar which needs to be very noticeable whenever there is an update.

**Comment by davidhedlund** [comment]
> This probably requires additional functionality such as an "update" button in the main toolbar which needs to be very noticeable whenever there is an update. Otherwise it would prevent many users from updating regularly.

Perhaps move the menu entry that I placed the mouse cursor with to a button?:

![image](https://github.com/RPCS3/rpcs3/assets/6416073/0746928e-1fb3-401a-80f9-00176c79afd6)

**Comment by Megamouse** [comment]
This already exists

> This probably requires additional functionality such as an "update" button in the main toolbar which needs to be very noticeable whenever there is an update. Otherwise it would prevent many users from updating regularly.

**Comment by Megamouse** [comment]
The reason why you're not supposed to disable the question is because updating a software should never be ignored, and neither should the software update on its own unless specificly told to.

**Comment by Megamouse** [comment]
Therefore it is a good thing that the option is hidden in a menu instead of shoved in your face.

**Comment by davidhedlund** [comment]
> The reason why you're not supposed to disable the question is because updating a software should never be ignored, and neither should the software update on its own unless specificly told to.

It's useful to hide the auto update dialogue box if you encounter regression bugs, and need to use an old version. If you are asked to press a yes button every time you open the emulator, there's a risk that you accidentally will update the system.

**Comment by FlexBy420** [comment]
If there is a known regression you won't be able to update to that broken build

**Comment by Asinin3** [comment]
I don't agree with making it easier for people to stop seeing updates. Yes there's a lot of updates, yes it can be a little annoying at times. But you know whats worse? Having all the hundreds of people asking for help with old builds, when the issue was already fixed.

Your regression linked can just be worked around by disabling audio buffering, that isn't a "broken" build, and it also only affects linux and some games.

**Comment by Megamouse** [comment]
This is basically implemented. And the pushback for the original request is noticable.
So I'll close this one.

**Comment by davidhedlund** [comment]
> This is basically implemented.

So rejected. As I described in the top post: "Check for updates on startup" exist, but it's moderately difficult to find it...

**Comment by davidhedlund** [comment]
Please close this as "Not planned" rather than as "Completed".

**Comment by davidhedlund** [comment]
Thanks.
